modules/swagger-codegen/src/main/java/io/swagger/codegen/InlineModelResolver.java

```diff
@@ -191,6 +191,16 @@ public void flatten(Swagger swagger) {
         }
     }
 
+    private void fixStringModel(ModelImpl m) {
+        if (m.getType() != null && m.getType().equals("string") && m.getExample() != null) {
+            String example = m.getExample().toString();
+            if (example.substring(0, 2).equals("\"") &&
+                    example.substring(example.length() - 2).equals("\"")) {
+                m.setExample(example.substring(2, example.length() - 2));
```

Comment:
Hmm, maybe I misunderstood what you are doing here ... I thought we have here a pair of `"` too much, which needs to be removed. (This is what the old `replace` code was doing, assuming there are no more quotation marks in the string.)
I thought the escaping (`"` → `\"`) was only done after this step (while outputting).

Anyway, if you want to remove not just a single `"`, but the combination `\"`, then you'll need to write it `"\\\""` as a Java string literal (the single backslash is escaping the `"`).

Though I'll try it out instead of just using my Java knowledge to discuss things theoretically.